python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -1935,6 +1945,7 @@ def convert_scale(g, op, block):
     scale = op.attr("scale")
     bias = op.attr("bias")
     bias_after_scale = op.attr("bias_after_scale")
+    act = op.attr("act")
```

Comment:
There's no "act" attribute in operator "scale". While we use `paddle.scale(x, act="relu")`, this will generate 2 op, include "scale" and "relu"